### PR TITLE
Add a users page that for now just lists users

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

**Comment by allysonjp715** [comment]
[Add page in UI for admins to assign roles to people](https://trello.com/c/zzLbt5u5/146-add-page-in-ui-for-admins-to-assign-roles-to-people)

File: verification/curator-service/ui/src/components/Users.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        config: {},
+        headers: {},
+    };
+    mockedAxios.get.mockImplementation(() => Promise.resolve(axiosResponse));
+    // Need to use shallow to wait for the component to mount.
+    var wrapper = shallow(<Users />);
+    const instance = wrapper.instance();
+    await instance.componentDidMount();
+    expect(mockedAxios.get).toHaveBeenCalledWith('/api/users');
+    expect(wrapper.text().includes('Alice Smith')).toBe(true);
```
Comment (axmb) [inline verification/curator-service/ui/src/components/Users.test.tsx:47]: Can we use `expect(wrapper.text())[.not].toContain($substring)`?

File: verification/curator-service/api/test/users.test.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import { User } from '../src/model/user';
+import app from '../src/index';
+import mongoose from 'mongoose';
+import request from 'supertest';
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+// Needs to create a new DB so it does not intefere with local logins
+const mongod = new MongoMemoryServer();
```
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/api/test/users.test.ts]: Nope, done.

File: verification/curator-service/ui/src/components/Users.test.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.test.tsx]: Done.

File: verification/curator-service/ui/src/components/Users.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        axios
+            .get<ListResponse>('/api/users')
+            .then((resp) => {
+                this.setState({ users: resp.data.users });
+            })
+            .catch((e) => {
+                this.setState({ users: [] });
+                console.error(e);
+            });
+    }
```
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.tsx:36]: Done

File: verification/curator-service/ui/src/components/Users.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        return (
+            <div>
+                {this.state.users.map(user =>
+                    <div key={user.googleID}>{user.name}</div>)}
+            </div>
+        )
+    }
+}
+
+export default Users;
```
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.tsx]: Lazy copy/pasting :) Done.

**Pull request merged**